studies/helpers.py

```diff
@@ -16,7 +15,14 @@
 
 @app.task
 def send_mail(
-    template_name, subject, to_addresses, cc=None, bcc=None, from_email=None, **context
+    template_name,
+    subject,
+    to_addresses,
+    cc=None,
+    bcc=None,
+    from_email=None,
+    reply_to=None,
+    **context,
 ):
```

Comment:
I *think* Black shouldn't be changing the formatting on this file if we're both using the same configuration from pre-commit. Is it running when you commit changes or are you having to run it manually? In the latter case it's possible it's a different version or with different preferences on line length etc.

Reply:
See commit https://github.com/lookit/lookit-api/pull/671/commits/4217b36baba62fe967657d1e51a2dfa9cd917d17 for my solution to this issue.